src/components/Photos.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import portland from '../../imgs/portland.jpg';
+import Austin from '../../imgs/Austin.jpg';
+
+
+function Photos() {
```

Comment:
```suggestion
function Photos({ photos }) {
```
`Photos` has to receive a `photos` prop that's being passed down by `Photoalbum`